environments/icds-new/postgresql.yml

```diff
@@ -24,9 +24,7 @@ override:
   pgbouncer_pool_mode: transaction
   pgbouncer_pool_timeout: 1
   pgbouncer_reserve_pool: 5
-  postgresql_effective_cache_size: 16GB
```

Comment:
Should we plan time/downtime to deploy to all postgres servers. I believe that this changes all servers configurations

Reply:
hmm, yes I see it will so let's schedule some downtime.